s4e-web/src/app/views/map-view/state/search-results/search-results.query.ts

```diff
@@ -0,0 +1,18 @@
+import {Injectable} from '@angular/core';
+import {QueryEntity} from '@datorama/akita';
+import {SearchResultsState, SearchResultsStore} from './search-results.store';
+import {SearchResult} from './search-result.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SearchResultsQuery extends QueryEntity<SearchResultsState, SearchResult> {
+
+  constructor(protected store: SearchResultsStore) {
+    super(store);
+  }
+
+  selectIsOpen() {
```

Comment:
Providing return type is a good practice. I know, that it can be inferred from the function call, but it's a good practice nevertheless